index.html

```diff
@@ -57,6 +57,174 @@ <h2>I`am a Software Developer</h2>
+              <ul>
+                <li class="list2">Html</li>
+                <li class="list2">Bootstrap</li>
+                <li class="list2">Ruby </li>
+              </ul>
+            </div><div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Website Protfolio</h1>
+              <h2>Website Protfolio</h2>
+              <p>Third A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul>
+                <li class="list2">Html</li>
+                <li class="list2">Bootstrap</li>
+                <li class="list2">Ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Profesional Art Printing Data</h1>
+              <h2>Profesional Art Printing Data</h2>
+              <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul><li class="list2">Html</li>
+                <li class="list2">Bootstrap</li>
+                <li class="list2">Ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+        <div class="work-2">
+          <div class="work2-content">
+            <h1>Profesional Art Printing Data</h1>
+            <h2>Profesional Art Printing Data</h2>
+            <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+            <ul><li class="list2">Html</li>
+              <li class="list2">Bootstrap</li>
+              <li class="list2">Ruby </li>
+            </ul>
+          </div>
+          <div class="work2-button"><button type="button">See Project</button></div>
+      </div>
+      <div class="work-2">
+        <div class="work2-content">
+          <h1>Website Protfolio</h1>
+          <h2>Website Protfolio</h2>
+          <p>Sixth A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+          <ul>
+            <li class="list2">Html</li>
+            <li class="list2">Bootstrap</li>
+            <li class="list2">Ruby </li>
+          </ul>
+        </div>
+        <div class="work2-button"><button type="button">See Project</button></div>
+      </div>
+        </div>
+          <div class="popUp-Container"></div>
+        </section>
+        <section class="about-me-section" id="about">
+          <div class="about-me-1">
+            <div class="about-me1-content">
+              <h1>About me</h1>
```

Comment:
- [ ] As mentioned earlier and the previous reviewer requested:pray:, it is essential to follow HTML best practices, including having only one `<h1>` heading in your document. I recommend checking the heading levels you have used and replacing this particular `<h1>` element with the appropriate heading level based on the content's hierarchy. For guidance on choosing the correct heading level, you can refer to the [HTML Heading Elements documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).

By ensuring the proper usage of heading elements, you will improve the structure and semantics of your HTML document.

Reply:
- [ ] Kindly change the tag of your <code>About me</code> to another heading other than <code>h1</code>, as requested by previous reviewers. It is best practice to have only one <code>h1</code> in a document.
For example, you can change it to :
```sh
<h2>About me</h2>
```
 🤗🤗🤗